first_ruby/.gitignore

```diff
@@ -0,0 +1,40 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```

Comment:
What is this file about?

Reply:
This will tell git which file to ignore.

Reply:
what you mean `ignore`?

Reply:
`ignore` in here means that git will not keep track of its changes